example_configs/wildfly-10.yaml

```diff
@@ -29,10 +31,11 @@ rules:
     attrNameSnakeCase: true
     name: wildfly_transaction_$1
 
-  - pattern: "^jboss.as<subsystem=undertow, server=(.+), http-listener=(.+)><>(bytes_.+|error_count|processing_time|request_count):"
+  - pattern: "^jboss.as<subsystem=undertow, server=(.+), (http[s]?-listener)=(.+)><>(bytes_.+|error_count|processing_time|request_count):"
     attrNameSnakeCase: true
-    name: wildfly_undertow_$3
+    name: wildfly_undertow_$4
     labels:
       server: $1
-      http_listener: $2
-
+      $2: $3
```

Comment:
The label names shouldn't vary here.

Reply:
According to the pattern there are only 2 choices for this label name: either `http_listener` or `https_listener.`
Bit if you prefer 2 separate pattern with fixed label names I will separate them.

Reply:
Why do you want two different label names here?

Reply:
I might be wrong but it looks for me as it is possible to have both http_listener and https_listener. If this is so we will have a clash with the same metric name and labels names.

Reply:
On my installation there was no http_listener at all, there was only an https_listener one

Reply:
That's something to distinguish with label values rather than names, and I'd presume both can't be on the same port.

Reply:
That's true, same names are not possible.
I've checked Wildfly config, the undertow system can contain 3 types of listeners: ajp listeners, http listeners and https listeners. That's why I see it as 3 separated (though living together) things. In this config only 2 of them are queried for metrics. That was a reason I added 2 different labels (via the pattern).
If you'd still prefer only `http_listener` as label, lets make it so. I've just shared my understanding :)

Reply:
Please, make final desision and I'll commit last changes

Reply:
Oh sorry, missed your response. It sounds like `listener` would be most correct here.

Reply:
changed `$2: $3` to `listener: $3`
So now all seems to be fine (: